react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -414,7 +419,7 @@ Axes.args = {
         viewports: [
             {
                 id: "view_1",
-                show3D: false,
+                show3D: true,
```

Comment:
Not sure why you changed this. I think it's good to have this example ortographic.

Reply:
Fixed